### PR TITLE
Refactor host vars from vagrant file to ansible folder

### PR DESCRIPTION
**Pull request by JamesGibo**

### CODE REVIEW

File: vagrant/provisioning/host_vars/auth [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+packages_to_install:
```
Comment (dannymeloy) [inline vagrant/provisioning/host_vars/auth]: In my never-ending quest to reduce this codebase down to a single line, could the common packages (nmos-common, mdnsbridge, etc) not be factored out into `defaults`, or an `all` group_vars file or something.. and then you should be able to just add the lists:
```
node_packages: "{{ standard_packages_somewhere_else + node_packages_above }}"
```

**Review by andrewbonney — COMMENTED** [review]
Works for me, and looks much cleaner

File: vagrant/provisioning/host_vars/auth (hunk 1 above)
Comment (JamesGibo) [reply vagrant/provisioning/host_vars/auth]: Done

**Review by dannymeloy — APPROVED** [review]
Legend. Much neater. If this all builds and tests fine then  LGTM

**Pull request merged**